### PR TITLE
make mongo client configurable

### PR DESCRIPTION
**Pull request by christian-steinmeyer**
Hey! I'd like to be able to configure the mongo client that's currently abstracted away by the ExperimentLoader. The easiest approach there would be to just forward a set of keyword arguments. What do you think?

### CODE REVIEW

**Comment by christian-steinmeyer** [comment]
Do I misinterpret the logs or is there some unrelated issue on CI?

File: incense/experiment_loader.py [hunk 1]
```diff
@@ -17,8 +17,8 @@
 class ExperimentLoader:
     """Loads artifacts related to experiments."""
 
-    def __init__(self, mongo_uri=None, db_name="sacred", unpickle: bool = True):
-        client: MongoClient = MongoClient(mongo_uri)
+    def __init__(self, mongo_uri=None, db_name="sacred", unpickle: bool = True, **docker_client_kwargs):
```
Comment (JarnoRFB) [inline incense/experiment_loader.py:20]:
```suggestion
    def __init__(self, mongo_uri=None, db_name="sacred", unpickle: bool = True, **mongo_client_kwargs):
```

Sounds more reasonable.

**Review by JarnoRFB — CHANGES_REQUESTED** [review]
Thanks for the contribution! I will check what the problem with the CI is.

File: incense/experiment_loader.py [hunk 2]
```diff
@@ -17,8 +17,8 @@
 class ExperimentLoader:
     """Loads artifacts related to experiments."""
 
-    def __init__(self, mongo_uri=None, db_name="sacred", unpickle: bool = True):
-        client: MongoClient = MongoClient(mongo_uri)
+    def __init__(self, mongo_uri=None, db_name="sacred", unpickle: bool = True, **mongo_client_kwargs):
+        client: MongoClient = MongoClient(mongo_uri, **docker_client_kwargs)
```
Comment (JarnoRFB) [inline incense/experiment_loader.py:21]:
```suggestion
        client: MongoClient = MongoClient(mongo_uri, **mongo_client_kwargs)
```

**Review by JarnoRFB — APPROVED** [review]

**Comment by JarnoRFB** [comment]
@christian-steinmeyer Sorry for the wait, fixed the travis issue now and merged the PR.

**Comment by JarnoRFB** [comment]
@christian-steinmeyer change was released in v0.0.15

**Comment by christian-steinmeyer** [comment]
Nice, thanks!

**Pull request merged**